ls1intum/Artemis
# `General`: Improve header responsiveness for instructors

### Checklist
#### General
#### Client

### Motivation and Context
The Manage Button taking the whole width of the windows when taking half of a 13,3 inch screen - which does not looked intended for me.
Also the separation between the `Show more` and the actual course description could be easier to spot.

### Description
- Always displaying the button to the right of the description and preventing the button to take the full screen width
- Adding arrows as indication of the expand / collapse direction

### Steps for Testing
Prerequisites:
- 1 Instructor
- 1 Student

1. `instructor`: Edit the parameters of a course _(icon, description, title)_
2. `student` / `instructor`: verify that the header still looks fine, resize the window
3. Click "Show more" and see that the arrow direction changes

### Testserver States
> [!NOTE]
> These badges show the state of the test servers.
> Green = Currently available, Red = Currently locked

![](https://byob.yarr.is/ls1intum/Artemis/artemis-test1)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test2)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test3)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test4)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test5)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test6)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test9)

### Review Progress
#### Code Review
#### Manual Tests

### Screenshots

#### New Version
https://github.com/ls1intum/Artemis/assets/63976129/ea9cabbd-8c31-410b-8ce3-115369dbf45c

#### Old Version


https://github.com/ls1intum/Artemis/assets/63976129/fb229858-1847-4599-916d-d663619fec86
